specs/Spec.Blake2.fst

```diff
@@ -397,20 +397,22 @@ let blake2_update1 a prev m i s =
 val blake2_update_last:
     a:alg
   -> prev:nat
-  -> m:bytes{prev + length m <= max_limb a}
+  -> rem:nat
+  -> m:bytes{prev + length m <= max_limb a /\ rem <= length m /\ rem <= size_block a}
   -> s:state a ->
   Tot (state a)
 
-let get_last (a:alg) (m:bytes) : block_s a =
-  let rem = length m % size_block a in
+let get_last_block (a:alg) (m:bytes) (rem:nat{rem <= length m /\ rem <= size_block a}) : block_s a =
```

Comment:
is there a way to rename this to make it apparent that the block is zero-padded?

Reply:
Good point: get_last_padded_block maybe?

Reply:
👍